nextcloudappstore/core/forms.py

```diff
@@ -50,27 +51,40 @@ class AppRegisterForm(Form):
                               'openssl base64</b>'))
     safe_help_fields = ['certificate', 'signature']
 
+def get_languages_local(language=None):
+    if language:
+        languages = [language]
+    else:
+        languages = [ l[0] for l in settings.LANGUAGES]
+
+    language_infos = []
+    for l in languages:
```

Comment:
[(i['code'], i['name_local']) for i in [get_language_info(l) for l in languages]]